src/bin/cybergrape/main.rs

```diff
@@ -26,25 +28,40 @@ fn main() {
     port.set_read_timeout(std::time::Duration::MAX)
         .expect("Failed to set read timeout");
 
+    let hdm = Rc::new(RefCell::new(Hdm::new()));
+    let mut accumulator = UpdateAccumulator::new(hdm.clone());
+
     // Read from the port and print the received data
     let mut buffer = [0; 256];
     let mut read_buf = Vec::new();
+
     loop {
         let read_len = port.read(&mut buffer).expect("Device disconnected");
 
         for &c in buffer.iter().take(read_len) {
             read_buf.push(c);
             if c == b'\n' {
                 match str::from_utf8(&read_buf) {
-                    Ok(s) => {
-                        println!("Received: {:?}", HardwareEvent::from_str(s));
```

Comment:
I think you would have said the same for me— do we want to turn this into a `warn!` or `debug!` log?

Reply:
these lines aren't actually in the PR...they are highlighted in red and have the little `-` on the right